src/base/b-virtual-scroll/README.md

```diff
@@ -0,0 +1,31 @@
+# base/b-virtual-scroll
+
+This module provides a component to render a large number of elements with Vue.
+
+### How to use
+
+```
+< b-virtual-scroll &
+	:dataProvider = 'demo.Pagination' |
+	:request = {get: {chunkSize: 12}} |
+	:option = 'b-card' |
+	:optionKey = (el, i) => resolveKey(el) |
+	:optionProps = getPropsForOption |
+	:dbConverter = convertDataToVirtual
+.
+```
+
+The `b-virtual-scroll` component always expects a specific data format, or rather,` db` must always be an object
```

Comment:
The component expects that loaded data will have the structure that matches with the `RemoteData` interface.

```typescript
export interface RemoteData extends Dictionary {
  /**
   * Data to render
   */
  data?: unknown[];

  /**
   * Total number of elements
   */
  total?: number;
}
```

You can use `dbConverter` to convert data to match this interface.

To specify what kind of component to render, you have to use the `option` property.
Mind, the property can be defined as a string or function.